### DISCUSSION TITLE
Como puedo restar meses a una fecha?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#362] [2. Questions and Answers] [answered]
Hola

Tengo la siguiente consulta: Resulta que tengo una componente de **Opción múltiple** de tipo "**listado de ítem**", cuyo Valor de cada ítem que se agrega corresponde a 3, 6, 12 respectivamente.

Con la ayuda de @Yiakman me indicó que con ($$TIME#month#X)|[date=>format=YYYY-MM-DDTHH:mm:ss], me arroja la fecha del mes que indico con X.

Entonces lo quiero es obtener la fecha que me marca el selector del listado de item, para luego usar el bot integración google calendar y dejar una cita con el resultado de la fecha.

Lo que tengo como code sería algo asi

$JOIN##(´´´($$TIME#month#´´´)#($VALUE#sentAnswer|data|[find=>identifier="campo"]|process|0)#(´´´)´´´)#(|[date=>format=YYYY-MM-DDTHH:mm:ss])

Me faltaría restar 1 en el resultado.

Gracias =>

**Comment by NicolasPLG** [comment]
Una de las formas que use, fue que restarlo en la componente de selección múltiple (si en el listado tengo, 3,6, 12 los code serían 2,5,11). Lo hice como respuesta rápida

**Comment by acedward** [comment, marked as answer]
¿Porque no usas un selector de fechas?
No entiendo bien en que formato tienes la data, en la cual quieres operar sobre

**Reply by NicolasPLG** [reply]
@acedward es un selector multiple cuyos string son: 3 meses, 6 meses y 12 meses, lo que hice fue en cada code colocar 2,5,11 respectivamente y asi al usar $$TIME#month#(Mes), obtenía el valor del mes que quería

**Reply by acedward** [reply]
si eso es lo que quieres hacer suena bien! 
¿Hay algo más para poder cerrar este issue?

**Reply by NicolasPLG** [reply]
hola @acedward lo cerraré, porque me sirvió la solución que indiqué.

Lo cerraré, gracias =>
